validator/src/main.rs

```diff
@@ -1772,16 +1772,26 @@ pub fn main() {
             None => ShredStorageType::default(),
             Some(shred_compaction_string) => match shred_compaction_string {
                 "level" => ShredStorageType::RocksLevel,
-                "fifo" => match matches.value_of("rocksdb_fifo_shred_storage_size") {
-                    None => ShredStorageType::rocks_fifo(default_fifo_shred_storage_size(
-                        &validator_config,
-                    )),
-                    Some(_) => ShredStorageType::rocks_fifo(Some(value_t_or_exit!(
-                        matches,
-                        "rocksdb_fifo_shred_storage_size",
-                        u64
-                    ))),
-                },
+                "fifo" => {
+                    warn!(
+                        "The value \"fifo\" for --rocksdb-shred-compaction has been deprecated. \
+                         Use of \"fifo\" will still work for now, but will be completely removed \
+                         in the future. To update, use \"level\" for --rocksdb-shred-compaction, \
```

Comment:
If we know which release we will remove, it might be nice to mention it here

Reply:
Fair enough. In the same vein of wanting confirmation about the BP for the change log, I would also want confirmation about the BP before putting specific numbers down.

Ideal would be to have deprecation warning show in v2.0.0, and hard error in v2.1.0

Reply:
Updated to state intention to remove in v2.1